packages/sdk/src/react/_internal/transaction-machine/execute-transaction.ts

```diff
@@ -52,6 +55,7 @@ export interface TransactionConfig {
 	chainId: string;
 	chains: readonly Chain[];
 	collectionAddress: string;
+	collectibleId?: string;
```

Comment:
This should not be part of the config; it should be in the exact method where you need it.

Imagine a table with multiple collectibles, now you would need to create a new class for every row. (instead of just using the same class, and just call the method with the collectibleId)